aws/src/main/resources/s3-signer-open-api.yaml

```diff
@@ -121,6 +121,10 @@ components:
           type: object
           additionalProperties:
             type: string
+        body:
+          type: string
+          description: Optional body of the S3 request to send to the signing API. This should only be populated
```

Comment:
Will update that the body serialization format should be JSON (otherwise it is under-spec'd).

Reply:
Actually, if we spec it to be JSON this forces a conversion on the client side for the DeleteObjects case (have to convert from XML to JSON). We probably don't want that burden on the client side and rather we want to send just the original format of the body to the server. Considering it's a relatively rarer case we actually need to send the body, that heavier conversion probably isn't needed. So I'm changing this back to just be "Optional body"